ansible/tasks/stickypedia.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: "create stickypedia user"
+  user:
+    name: "stickypedia"
+    shell: "usr/sbin/nologin"
+    home: "/var/www/stickypedia"
+    system: true
+
+block:
+  - name: "clone git with Stickypedia content"
+    git:
+      repo: {{ stickypedia.git_content }}
+      dest: "/var/www/stickypedia/stickypedia.{{ canonical_hostname }}"
+      version: "master"
+    diff: false
+
+  - name: "install prerequisites for gollum"
+    apt:
+      name:
+        - "ruby"
+        - "ruby-dev"
+        - "make"
+        - "zlib1g-dev"
+        - "libicu-dev"
+        - "build-essential"
+        - "git"
+        - "cmake"
+      state: "present"
+    become: true
+
+  - name: "install gem gollum"
+    gem:
+      name: gollum
+      state: latest
```

Comment:
We might want to pin the version here, and not whatever version is the newest. See rubygems.org for the version, the command would then be `gem install gollum -v whatever`

Reply:
pinned to version 4.1.4